dialogue-clients/src/main/java/com/palantir/dialogue/clients/ReloadingClientFactory.java

```diff
@@ -170,27 +229,37 @@ public PerHostClientFactory perHost(String serviceName) {
         Preconditions.checkNotNull(serviceName, "serviceName");
         String channelName = ChannelNames.reloading(serviceName, params);
 
-        Refreshable<List<DialogueChannel>> perHostDialogueChannels = params.scb()
+        Refreshable<List<DialogueChannel>> perHostDialogueChannels = getServiceRefreshable(serviceName)
```

Comment:
Updated to handle `perHost` here, the only remaining caller into the legacy `getNonReloadingChannel` over is the non-reloading method which takes a fully realized ServiceConfiguration as input. We can handle that in a separate PR, as well as the `getNonReloading(Class<T> clazz, ClientConfiguration clientConf)` overload